dartagnan/src/main/java/com/dat3m/dartagnan/utils/GitInfo.java

```diff
@@ -16,11 +16,14 @@ public class GitInfo {
 	static Properties properties = new Properties();
 	
 	public static void CreateGitInfo() throws IOException {
-        try (InputStream is = Dartagnan.class.getClassLoader()
+       try (InputStream is = Dartagnan.class.getClassLoader()
                 .getResourceAsStream("git.properties")) {
-            properties.load(is);
-            logger.info("Git branch: " + properties.getProperty("git.branch"));
-            logger.info("Git commit ID: " + properties.getProperty("git.commit.id"));
+		   if (is == null) {
+			   return;
+		   }
+		   properties.load(is);
+		   logger.info("Git branch: " + properties.getProperty("git.branch"));
+		   logger.info("Git commit ID: " + properties.getProperty("git.commit.id"));
```

Comment:
Does this fix the annoying issue that every now and then after compiling you get a `null stream` (or something of the sort) exception? Have you seen this? I mostly got it when I was using a proxy and I had some problem on my proxy configuration

Reply:
Yes, I got null streams sometimes when running from the IDE I believe. Since it didn't seem to be crucial output, I just added the simple null-check-and-return.